Carbon.AccountManagement/Carbon.AccountManagement.psd1

```diff
@@ -18,7 +18,7 @@
     RootModule = 'Carbon.AccountManagement.psm1'
 
     # Version number of this module.
-    ModuleVersion = '0.0.0'
+    ModuleVersion = '1.0.0'
 
     # ID used to uniquely identify this module
     GUID = ''

```

Comment:
Make sure the `CompatiblePSEditions` property is set to `@( 'Desktop', 'Core' )`.